lib/travis/api/v3/queries/enterprise_license.rb

```diff
@@ -0,0 +1,13 @@
+module Travis::API::V3
+  class Queries::EnterpriseLicense < Query
+    def active_users(expiration_time)
+      this_year = Date.parse(expiration_time)
+      last_year = this_year - 365
+
+      Models::Email
+      .joins("INNER JOIN commits c ON LOWER(c.committer_email) = LOWER(emails.email)")
```

Comment:
We may need to consider indexes for this query, as there will likely be many rows in the commits table.